hal/src/xenon/pinmap_defines.h

```diff
@@ -87,85 +87,3 @@
 #define ANTSW2 30
 
 #endif // PLATFORM_ID == PLATFORM_XENON
```

Comment:
We can probably remove this macro condition. Same header file with regard to the Argon platform.